src/app/help/faq/faq.component.html

```diff
@@ -32,52 +40,56 @@ <h1>Frequently Asked Questions</h1>
-                <a href="javascript:void(0)" tabindex="0" (click)="scroll('mostViewTitle')">Most Viewed FAQs</a>
+                <a href="javascript:void(0)" tabindex="0" (click)="scroll('mostViewTitle')" class="sidenav-link">Most Viewed FAQs</a>
               </li>
               <li class="usa-sidenav__item">
-                <a href="javascript:void(0)" tabindex="0" (click)="scroll('backgroundInfoTitle')">Background</a>
+                <a href="javascript:void(0)" tabindex="0" (click)="scroll('backgroundInfoTitle')" class="sidenav-link">Background</a>
               </li>
               <li class="usa-sidenav__item">
-                <a href="javascript:void(0)" tabindex="0" (click)="scroll('aboutTitle')">About SRT</a>
+                <a href="javascript:void(0)" tabindex="0" (click)="scroll('aboutTitle')" class="sidenav-link">About SRT</a>
               </li>
               <li class="usa-sidenav__item">
-                <a href="javascript:void(0)" tabindex="0" (click)="scroll('scanningProcessTitle')">Explore the Scanning Process</a>
+                <a href="javascript:void(0)" tabindex="0" (click)="scroll('scanningProcessTitle')" class="sidenav-link">Explore the Scanning Process</a>
               </li>
               <li class="usa-sidenav__item">
-                <a href="javascript:void(0)" tabindex="0" (click)="scroll('manageWorkloadTitle')">Manage/ Review Solicitations</a>
+                <a href="javascript:void(0)" tabindex="0" (click)="scroll('manageWorkloadTitle')" class="sidenav-link">Manage/ Review Solicitations</a>
               </li>
               <li class="usa-sidenav__item">
-                <a href="javascript:void(0)" tabindex="0" (click)="scroll('requestSupportTitle')">Request Support</a>
+                <a href="javascript:void(0)" tabindex="0" (click)="scroll('requestSupportTitle')" class="sidenav-link">Request Support</a>
               </li>
             </ul>
           </nav>
 
           <!-- FAQ Content Area -->
           <div class="grid-col-9">
-            <div class="usa-accordion">
-              <div *ngFor="let section of faq; let sectionIndex = index">
-                <h2 class="search-title-header" id="{{section.title}}">{{section.name}}</h2>
-                <div *ngIf="section.children">
-                  <div *ngFor="let element of section.children; let i = index">
-                    <!-- Accordion Button with + or - based on expansion state -->
-                    <button class="usa-accordion__button" aria-expanded="{{ isExpanded(section.id + i) }}" aria-controls="faq-{{ section.id + i }}" (click)="toggleAccordion(section.id + i)">
-                      <span class="accordion-symbol">{{ isExpanded(section.id + i) ? '-' : '+' }}</span>
-                      {{ element.title }}
+            <div class="usa-accordion usa-accordion--bordered">
+              <div *ngFor="let section of faq; let sectionIndex = index" class="usa-accordion__item mb-8">
```

Comment:
We the padding between the individual FAQ sections. Not the section.children items.